test/internal/datapath/datapath_win.go

```diff
@@ -0,0 +1,178 @@
+package datapath
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-container-networking/test/internal/k8sutils"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	apiv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+)
+
+func podTest(ctx context.Context, clientset *kubernetes.Clientset, srcPod *apiv1.Pod, cmd []string, rc *restclient.Config, passFunc func(string) error) error {
+	logrus.Infof("podTest() - %v %v", srcPod.Name, cmd)
+	output, err := k8sutils.ExecCmdOnPod(ctx, clientset, srcPod.Namespace, srcPod.Name, cmd, rc)
+	if err != nil {
+		return errors.Wrapf(err, "failed to execute command on pod: %v", srcPod.Name)
+	}
+	return passFunc(string(output))
+}
+
+func WindowsPodToPodPingTestSameNode(ctx context.Context, clientset *kubernetes.Clientset, nodeName, podNamespace, labelSelector string, rc *restclient.Config) error {
+	logrus.Infof("Get Pods for Node: %s", nodeName)
+	pods, err := k8sutils.GetPodsByNode(ctx, clientset, podNamespace, labelSelector, nodeName)
+	if err != nil {
+		logrus.Error(err)
+		return err
+	}
+	if len(pods.Items) <= 1 {
+		return errors.New("Less than 2 pods on node")
+	}
+
+	// Get first pod on this node
+	firstPod, err := clientset.CoreV1().Pods(podNamespace).Get(ctx, pods.Items[0].Name, metav1.GetOptions{})
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("Getting pod %s failed with %v", firstPod.Name, err))
+	}
+	logrus.Infof("First pod: %v", firstPod.Name)
```

Comment:
Can we also print this pod IP addresses here (ipv4 && ipv6) so that we can know source and destination IP that's more clear.
I see you print ip address on second pod below: 
logrus.Infof("Second pod: %v %v", secondPod.Name, secondPod.Status.PodIP)